nixos/modules/virtualisation/vmware-guest.nix

```diff
@@ -37,6 +37,23 @@ in
         serviceConfig.ExecStart = "${open-vm-tools}/bin/vmtoolsd";
       };
 
+    # Mount the vmblock for drag-and-drop and copy-and-paste.
+    systemd.mounts = [
+      {
+        description = "VMware vmblock fuse mount";
+        documentation = [ "https://github.com/vmware/open-vm-tools/blob/master/open-vm-tools/vmblock-fuse/design.txt" ];
+        before = [ "umounts.service" "vmware.service" ];
```

Comment:
Why umounts.service
thats a target and its added by default to before https://www.freedesktop.org/software/systemd/man/systemd.mount.html#Default%20Dependencies

Reply:
Good catch. This is based on https://kb.vmware.com/s/article/74671 and I have been running it locally for some time.